src/pages/index.tsx

```diff
@@ -120,6 +150,14 @@ const HomePage: NextPageWithLayout = () => {
             onOpenChange: handleModalCloseOnEsc,
           }}
         />
+        <VmComponent
+          // src={components.nearOrg.notifications.iosHomeScreenAlert}
+          src="dima_sheleg.near/widget/NearOrg.Notifications.HomeScreenAlert"
```

Comment:
To test it locally use `bos-loader` and change this `src`

Reply:
Let's [merge this](https://github.com/near/near-discovery-components/pull/367) to facilitate testing.